### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by kevin-10**
- Added proper secondary skills for bushido/ninjitsu/poisoning weapon specials
- added Mocapotl's Obsidian Sword to shadow guard roof encounter
- removed gems leaking to internal map
- fixed focus mob for dinosaurs
- fixed container stacking, requires Core recompile!!!

### CODE REVIEW

**Comment by Argalep** [comment]
* We need to delete the unnecessary MocapotilsObsidianSword

`public class MocapotilsObsidianSword : StoneWarSword
	{`

TimeOfLegends.cs
* We have to add MocapotlsObsidianSword to the m_PigmentList

File: Scripts/Services/Expansions/Time Of Legends/Shadowguard/Artifacts.cs [hunk 1]
```diff
@@ -1016,4 +1016,41 @@ public override void Deserialize(GenericReader reader)
             int version = reader.ReadInt();
         }
     }
+
+    public class MocapotlsObsidianSword : PaladinSword
```
Comment (Argalep) [inline Scripts/Services/Expansions/Time Of Legends/Shadowguard/Artifacts.cs]: PaladinSword could not be found

Comment (Argalep) [inline Scripts/Services/Expansions/Time Of Legends/Shadowguard/Artifacts.cs]: [PaladinSword.zip](https://github.com/ServUO/ServUO/files/1054113/PaladinSword.zip)

File: Scripts/Services/Expansions/Time Of Legends/Shadowguard/Artifacts.cs [hunk 2]
```diff
@@ -1016,4 +1016,41 @@ public override void Deserialize(GenericReader reader)
...
+
+        [Constructable]
+        public MocapotlsObsidianSword()
+        {
+            WeaponAttributes.HitHarm = 50;
+            WeaponAttributes.HitPhysicalArea = 50;
+            WeaponAttributes.HitLeechStam = 100;
+            WeaponAttributes.SplinteringWeapon = 20;
+            Attributes.WeaponDamage = 75;
+
```
Comment (Argalep) [inline Scripts/Services/Expansions/Time Of Legends/Shadowguard/Artifacts.cs]: Missind Attr.

Attributes.WeaponSpeed = 40;

File: Scripts/Scripts.csproj [hunk 3]
```diff
@@ -772,6 +772,7 @@
     <Compile Include="Items\Artifacts\Decorative\BaseDecorationArtifact.cs">
       <SubType>Code</SubType>
     </Compile>
+    <Compile Include="Items\Equipment\Weapons\PaladinSword.cs" />
```
Comment (Argalep) [inline Scripts/Scripts.csproj]: The file is not in the project

Comment (kevin-10) [inline Scripts/Scripts.csproj]: Crapola!

**Pull request merged**